apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1377,15 +1394,29 @@ def _ingest_uns_node(
             # This is a structured array, which we do not support.
             logging.log_io(msg, msg)
             return
-        _ingest_uns_ndarray(
-            coll,
-            key,
-            value,
-            platform_config,
-            context=context,
-            use_relative_uri=use_relative_uri,
-            ingestion_params=ingestion_params,
-        )
+
+        if value.dtype.char == "U" or value.dtype.char == "O":
+            # In the wild it's quite common to see arrays of strings in uns data.
```

Comment:
Is it necessary to check if all the entries in an `O` are strings (or None), or is this an assumption we can make?

Reply:
@thetorpedodog this is an assumption we can make